# HG changeset patch
# User jonnenauha
# Date 1406003802 -10800
#      Tue Jul 22 07:36:42 2014 +0300
# Branch v1-9
# Node ID d6fb78a77281a0d5e6b82d2bfcfccbcbf160be1b
# Parent  47adc0bab3931dd2802d99382d537458165ec0b2
Enable headless renderer from https://github.com/jonnenauha/ogre-headless-renderer

diff -r 47adc0bab393 -r d6fb78a77281 RenderSystems/CMakeLists.txt
--- a/RenderSystems/CMakeLists.txt	Thu Sep 12 18:32:30 2013 +0300
+++ b/RenderSystems/CMakeLists.txt	Tue Jul 22 07:36:42 2014 +0300
@@ -73,3 +73,7 @@
 if (OGRE_BUILD_RENDERSYSTEM_STAGE3D AND FLASHCC)
     add_subdirectory(Stage3D)
 endif()
+
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/Headless)
+    add_subdirectory (Headless)
+endif ()
